readme.txt

```diff
@@ -93,6 +93,9 @@ Absolutely! We offer a 14-day free trial on every plan. No obligation. No credit
 
 == Changelog ==
 
+= 1.7.18 =
+* Compatibility update: Tested with Woocommerce versions up to 4.5.1
```

Comment:
Add wordpress version in this message as well.

Reply:
Updated the message. Just a hint that the wordpress version is mentioned in the same file header row 5: Tested up to: 5.5.1

Reply:
Yeah, I know, but we're trying to make it as explicit as possible.